# These are the parameters for 
# the RC Commander. Middle value is measured when throttle stick is untouched.
# These values can be obtained by rostopic echo /rc_command
max_throttle_output: 1936.0
min_throttle_output: 996.0
middle_throttle_output: 1426.0

max_steering_output: 1636.0
min_steering_output: 1600.0
middle_steering_output: 1616.0

# Max and min linear speed of the car.
# Min value is the minimum throttle value that car starts to move
max_car_linear_speed: 35.0
min_car_linear_speed: 0.0

# Angle limits for the car.
max_car_angle: 20.0
min_car_angle: 0.0

#There are different modes of RACECAR Simulator.
#These parameters are for choosing between them√ß
unique_command: true
stamped_output: true

# This is the topic RACECAR Simulator expects the ackermann_msgs
drive_topic: '/drive'

# This is the topic which Arduino publishes the RC command measurements
rc_topic: '/rc_command'